src/main/java/spoon/reflect/declaration/CtSimpleType.java

```diff
@@ -31,7 +32,7 @@
  * @param <T>
  *            the actual runtime type
  */
-public interface CtSimpleType<T> extends CtNamedElement, CtTypeMember {
+public interface CtSimpleType<T> extends CtNamedElement, CtTypeInformation, CtTypeMember {
```

Comment:
Mh. I'm not sure it is really a problem but now, a `CtSimpleType` can access to its super class, super interfaces and can calculate if it is a sub type of another type.